com.unity.render-pipelines.universal/Runtime/2D/Renderer2D.cs

```diff
@@ -162,7 +162,8 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
                 {
                     stackHasPostProcess = stackHasPostProcess && DebugHandler.IsPostProcessingAllowed;
                 }
-                DebugHandler.Setup(context, ref cameraData);
+
+                DebugHandler.Setup(context, ref cameraData, renderingData.commandBuffer);
```

Comment:
should we pass RenderingData instead of CommandBuffer for consistency with other methods? Also passing RenderingData should be more future proof in case more info (already in renderingData) would be required in the future by the function

Reply:
Yeah, i initially did this by passing the whole RenderingData, but then i thought that it's probably not necessary just for a single member of a struct. I thought that if there was a need for more info from renderingData - we could swap this then. I'm happy to change this to RenderingData if you think it's better that way

Reply:
I did the same on my original prototype and I had this suggestion from Erik, so both are fine, but in general passing RenderingData is considered less restrictive

Reply:
sure. will change these calls to pass RendererData then

Reply:
fixed in 3939959